src/status_im/contexts/wallet/sheets/network_selection/view.cljs

```diff
@@ -0,0 +1,79 @@
+                                                  {:currency currency
+                                                   :balance  total-balance
+                                                   :token    token})
+        crypto-formatted                         (utils/get-standard-crypto-format token total-balance)
+        fiat-formatted                           (utils/get-standard-fiat-format crypto-formatted
+                                                                                 currency-symbol
+                                                                                 fiat-value)
+        token-symbol                             (:symbol token)
+        mainnet?                                 (= network-name :mainnet)]
+    [rn/view {:style (style/network-list-container mainnet?)}
+     [quo/network-list
+      {:label         (name network-name)
+       :network-image (quo.resources/get-network network-name)
+       :token-value   (str crypto-formatted " " token-symbol)
+       :fiat-value    fiat-formatted
+       :on-press      #(on-select-network network)}]]))
+
+(defn view
+  [{:keys [token on-select-network]}]
+  (let [{token-networks :networks
+         token-symbol   :symbol} token
+        mainnet-network          (first
+                                  (filter
+                                   (fn [network]
+                                     (= (:network-name network) :mainnet))
+                                   token-networks))
+        layer-2-networks         (filter
+                                  (fn [network]
+                                    (not= (:network-name network) :mainnet))
+                                  token-networks)
+        currency                 (rf/sub [:profile/currency])
+        currency-symbol          (rf/sub [:profile/currency-symbol])]
+    [:<>
+     [rn/view {:style style/header-container}
+      [quo/text
+       {:size   :heading-2
+        :weight :semi-bold}
+       (i18n/label :t/select-network)]
+      [quo/context-tag
+       {:type            :token
+        :size            24
+        :token           token-symbol
+        :container-style style/context-tag}]]
+     (when mainnet-network
+       [network-item
+        {:network           mainnet-network
+         :token             token
+         :on-select-network on-select-network
+         :currency          currency
+         :currency-symbol   currency-symbol}])
+     [quo/divider-label {:container-style style/divider-label}
+      (i18n/label :t/layer-2)]
+     [rn/flat-list
+      {:data      (vec layer-2-networks)
+       :render-fn (fn [network]
+                    [network-item
+                     {:network           network
+                      :token             token
+                      :on-select-network on-select-network
+                      :currency          currency
+                      :currency-symbol   currency-symbol}])}]]))
```

Comment:
We should avoid using anonymous functions and take advantage of `use-callback`.

Reply:
Fixed